### ISSUE TITLE
[heartbeat] Add Bot Latency as Dynamic Parameter in Heartbeat URL

### ISSUE THREAD

**Issue by Ceddicedced** [PhasecoreX/PCXCogs#160]
Hello hello,

I just stumbled across uptime kuma. While migrating to it is noticed that it supports a push monitor. While searching for supporting cogs I came back to your repo ;)

Another thing I noticed is that this push monitor by uptime kuma supports multiple tags as seen here: [Push Issue](https://github.com/louislam/uptime-kuma/issues/279#issuecomment-1356352436 ). 

Maybe you (or myself) could add support for setting the parameter in a more dynamic way. E.g. the up or down could be set depending of the bots status to the discord server ( as the bot can run even when not reaching the discord server). Even more interesting would be setting the "ping" tag acording to `self.bot.latency`.

Tell me what you think about this ✌️

**Comment by PhasecoreX** [comment]
Yeah, I can add the ping as a variable `{{ping}}` most likely. I am not sure about the up/down status part though, not sure where to find that in the bot data. You have any idea on that?

**Comment by PhasecoreX** [comment]
I have added `{{ping}}`. Let me know if it works correctly, since I don't have a way of testing that out.

**Comment by Ceddicedced** [comment]
Works fine!
